monai/data/meta_tensor.py

```diff
@@ -328,6 +328,51 @@ def as_tensor(self) -> torch.Tensor:
         """
         return self.as_subclass(torch.Tensor)  # type: ignore
 
+    def get_array(self, output_type=np.ndarray, dtype=None, *_args, **_kwargs):
```

Comment:
Could you please add the typehints referring to:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/utils/type_conversion.py#L232

Thanks.